nano/js/nano_state_default.js

```diff
@@ -4,11 +4,11 @@ var NanoStateDefault = new NanoStateDefaultClass();
 
 function NanoStateDefaultClass() {
 
-    this.key = 'default';
+	this.key = 'default';
```

Comment:
Got indented one tab forwards (other lines lower down the file as well)

Reply:
No, it didn't. This is because GitHub displays tabs as 8 spaces wide.